### DISCUSSION TITLE
Full size viewport

### DISCUSSION THREAD

**Discussion by stychu** [cypress-io/cypress#17444] [General]
Hi,
Im getting annoyed really hard by the empty spaces inside the viewport when working on Cypress tests.
Default values are 1000x660
Can i somehow make it always fill the whole space of viewport without setting any fixed values?

<img width="1766" alt="Screenshot 2021-07-22 at 13 08 16" src="https://user-images.githubusercontent.com/9383657/126630150-29f353fe-6faf-403f-948c-2dd4d4e0c8cf.png">

**Comment by AmirAmArA** [comment]
Hey there @stychu , 

to override default viewport you can add it in your config file;

for example: 

`
export default defineConfig({
 viewportHeight:720,
  viewportWidth:1280,
  }`
